sklearn/neighbors/_graph.py

```diff
@@ -230,6 +230,10 @@ class KNeighborsTransformer(
 
     The transformed data is a sparse graph as returned by kneighbors_graph.
 
+    See
+    :ref:`sphx_glr_auto_examples_neighbors_approximate_nearest_neighbors.py`
+    for an example on how to use the API.
+
```

Comment:
This should also go in TSNE's docstring. And probably in a "Notes" section instead, in both places.

```suggestion
    For an example of using `KNeighborsTransformer` in combination with
    :class:`~sklearn.manifold.TSNE` see
    :ref:`sphx_glr_auto_examples_neighbors_approximate_nearest_neighbors.py`.
```